packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -452,6 +449,29 @@ const toggleOrientation = async ( driver ) => {
 	}
 };
 
+const isEditorVisible = async ( driver, iteration = 0 )  => {
+	if ( iteration >= 25 ) {
+		throw new Error(
+			`Gutenberg Editor is still not visible after ${ iteration } retries!`
+		);
+	} else if ( iteration !== 0 ) {
+		// wait 1 second before trying to locate element again
+		await driver.sleep( 1000 );
+	}
+
+	const blockLocator = isAndroid()
+		? `//android.widget.EditText[@content-desc="Post title. Welcome to Gutenberg!"]`
+		: `(//XCUIElementTypeScrollView/XCUIElementTypeOther/XCUIElementTypeOther[@name="Post title. Welcome to Gutenberg!"])`;
```

Comment:
AFAIK this element will be present in all tests, as by default posts always have a title. However, I'm a bit concerned about having the text `Welcome to Gutenberg!` in the query because it might change in the future. Would it be a way to do a partial query with only `Post title.`?

Reply:
Yup that's a good point, the reason I used the full text was so that the number of elements returned is always 1. I tried using partial query in a previous commit and on Android it returned 2 elements (on iOS, 6). I'll check if there are other locators I can use for this.